src/main/java/com/groupseven/sysc4806project/ShoppingCartService.java

```diff
@@ -0,0 +1,109 @@
+package com.groupseven.sysc4806project;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.*;
+
+@RestController
+@RequestMapping("/api/carts")
+public class ShoppingCartService {
+
+
+    private final ShoppingCartRepository shoppingCartRepository;
+    @Autowired
+    private BookRepository bookRepository;
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    public ShoppingCartService (ShoppingCartRepository shoppingCartRepository){
+        this.shoppingCartRepository = shoppingCartRepository;
+    }
+
+    @GetMapping("/{cartId}")
+    public ShoppingCart getCart(@PathVariable int cartId){
+        return shoppingCartRepository.findById(cartId).orElse(null);
+    }
+
+    @GetMapping("/create/{userId}")
+    public Integer createCart(@PathVariable int userId){
```

Comment:
We shouldn't be allowed to create multiple carts for the same user. I would suggest returning the cart ID of the user's cart if one already exists